plugins/header_rewrite/operators.cc

```diff
@@ -1193,6 +1193,57 @@ OperatorSetHttpCntl::exec(const Resources &res) const
   return true;
 }
 
+void
+OperatorSetPluginCntl::initialize(Parser &p)
+{
+  Operator::initialize(p);
+  const std::string &name  = p.get_arg();
+  const std::string &value = p.get_value();
+
+  if (name == "TIMEZONE") {
```

Comment:
`LOCAL` and `GMT` look like the only two possible values, should add that to the docs.

Reply:
It's already written on the docs.